### ISSUE TITLE
Make the title an actual ##title

### ISSUE THREAD

**Issue by igorsantos07** [MelleD/lovelace-expander-card#217]
**Is your feature request related to a problem? Please describe.**
Currently, the title field is simply a line of small text.

**Describe the solution you'd like**
It should align with the title markup for the other default HA cards.

**Describe alternatives you've considered**
Right now, I'm using a markdown card as title-card, with an `h2` (e.g. `## Here be title`).

**Additional context**
To avoid breaking for current users, I guess an option should be included to "make title bigger" or something like that - although I do think this should be the default, to keep in line with the rest of the cards.

And god, thanks for resurrecting the repo from the original developer! <3

**Comment by MelleD** [comment]
Hey @igorsantos07 ,

you have currently 2 possibility to achieve that:
a) Use card mod e.g.
```
        card_mod:
          style: |
            button.header {
              font-size: 30px;
            }
```
b) Use your markdown card as title card. I make the same thing with mushroom template cards to styling my header for the expander

Please let me know if this is working for you

**Comment by igorsantos07** [comment]
Yeah, I know there are options around it, as I even mentioned in the description. However, I still think it would be saner if the title did look like the default card titles from HA... Either by default or via a simple toggle option, instead of having to resort to `card-mod` or a secondary card _just to make the title look like a title_.

**Comment by MelleD** [comment]
I just wanted to show you alternatives to achieve your goal until someone contributes the feature, but of course you can wait :-). or even better, do a PR straight away

**Comment by igorsantos07** [comment]
There are a lot of stuff in HA I want to make PRs, but (1) no clue where to start with testing HACS stuff on my instance and, most importantly, (2) got a huge backlog and attention deficit to battle at work 🥲

**Comment by MelleD** [comment]
I think everyone has the same situation, where and when do you invest valuable time. I don't think anyone has too much time.

**Comment by MelleD** [comment]
I think there are already some very good cards for homework that can be used for styling, including mushroom title cards, mushroom template cards, and markdown cards.

I think it makes a lot of sense to use existing cards. For this reason, I'm inclined to close this ticket.

**Comment by dcapslock** [comment]
Once PR #760 is merged you can use `style` to directly style `.title` (or button.header as per card-mod example). Card-mod no longer required.

```yaml
style: |
  .title {
    font-size: var(--ha-font-size-l);
  }
```

As for using a Frontend header, you can use `title-card` with `type: header` and get all the benefits of that card like adding badges. So with `title-card` option and directly `style` you have many options.

title-card header example

```yaml
title-card:
  type: heading
  heading: Title
  heading_style: title
  badges:
    - type: entity
      show_name: false
      show_state: true
      show_icon: true
      entity: light.bed_light
  icon: mdi:account
```
